buildSrc/src/main/java/net/neoforged/neodev/installer/LibraryCollector.java

```diff
@@ -37,11 +40,19 @@ public static List<Library> resolveLibraries(List<URI> repositoryUrls, Collectio
         var result = collector.libraries.stream().map(future -> {
             try {
                 return future.get();
+            } catch (ExecutionException e) {
```

Comment:
I am curious as to why this is better than just throwing a new runtime exception in all cases?

Reply:
Yes, remove it. Either we do everywhere or we do it nowhere.